README.md

```diff
@@ -22,4 +23,53 @@ Initialize the packages using:
 * `python -m pip install .`
 * `python -m pip install .[test]`
 
-Run `pytest` to run all tests in `/tests`
\ No newline at end of file
+Run `pytest` to run all tests in `/tests`
+
+## Status Board
+
+Here are the BICAN LinkML knowledgebase schemas and their statuses.
+
+| Model | Version | Release | Status |
+|:--|:--|:--|:--|
+| [BICAN BioLink] | [] |  [] | under development |
+| [CCN2] | [] | [] | under development |
+| [KB Model] | [] | [] | under development |
+| [Purple Boxes] | [] | [] | under development |
```

Comment:
perhaps we should call this something else, and make everything under KB Model.

Reply:
I'll change the name here for now. We can change the path or remove this when we make a release.